socs/agents/hwp_gripper/agent.py

```diff
@@ -170,7 +170,6 @@ def brake(self, session, params=None):
 
     @ocs_agent.param('mode', default='push', type=str, choices=['push', 'pos'])
     @ocs_agent.param('actuator', default=1, type=int, check=lambda x: 1 <= x <= 3)
-    @ocs_agent.param('distance', default=0, type=float, check=lambda x: -10. <= x <= 10.)
```

Comment:
We still need most of this, it should just be:
```
@ocs_agent.param('distance', default=0, type=float)
```

Reply:
Oh, sorry I will fix it.